File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/maps/GoogleMapsActivity.java
package com.jdroid.android.sample.ui.google.maps;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class GoogleMapsActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return GoogleMapsFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/maps/GoogleMapsActivity.kt
package com.jdroid.android.sample.ui.google.maps

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class GoogleMapsActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return GoogleMapsFragment::class.java
    }
}
